libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -400,7 +398,12 @@ bool AP_Arming::battery_checks(bool report)
             return false;
         }
 
-        for (int i = 0; i < _battery.num_instances(); i++) {
+        // all following checks are bypassed if USB is connected
+        if (hal.gpio->usb_connected()) {
```

Comment:
Isn't there a parameter that flags if we should be enabling or disabling the USB opt outs?